pkg/kdump/kdump-view.jsx

```diff
@@ -456,24 +450,28 @@ export class KdumpPage extends React.Component {
                 const tooltip = _("No memory reserved. Append a crashkernel option to the kernel command line (e.g. in /etc/default/grub) to reserve memory at boot time. Example: crashkernel=512M");
                 serviceHint = (
                     <Tooltip id="tip-service" content={tooltip} position={TooltipPosition.bottom}>
-                        <span className="popover-ct-kdump fa fa-lg fa-info-circle" />
+                        <OutlinedQuestionCircleIcon className="popover-ct-kdump" />
                     </Tooltip>
                 );
             }
-            kdumpServiceDetails = <button role="link" className="popover-ct-kdump link-button" onClick={this.handleServiceDetailsClick}>{serviceDescription}{serviceHint}</button>;
+            kdumpServiceDetails = (
+                <>
+                    <Button variant="link" isInline className="service-link-ct-kdump" onClick={this.handleServiceDetailsClick}>{serviceDescription}</Button>
+                    {serviceHint}
+                </>
+            );
         } else if (this.props.kdumpStatus && !this.props.kdumpStatus.installed) {
             const tooltip = _("Kdump service not installed. Please ensure package kexec-tools is installed.");
+            // FIXME: Accessibility needs to be improved: https://github.com/patternfly/patternfly-react/issues/5535
             kdumpServiceDetails = (
                 <Tooltip id="tip-service" content={tooltip} position={TooltipPosition.bottom}>
-                    <button className="popover-ct-kdump link-button">
-                        <span className="fa fa-lg fa-info-circle" />
-                    </button>
+                    <OutlinedQuestionCircleIcon className="popover-ct-kdump" />
                 </Tooltip>
             );
         }
         var serviceWaiting;
         if (this.props.stateChanging)
-            serviceWaiting = <div className="spinner spinner-sm" />;
+            serviceWaiting = <Spinner size="md" />;
```

Comment:
This is (in my option) misaligned:
![aligningcoredumps](https://user-images.githubusercontent.com/12330670/110837249-4726c200-82a1-11eb-8bbf-8a565db51628.png)

Both spinner and text are top aligned with switch, which I don't think is right.

Reply:
Well spotted, I didn't notice that yesterday. That misalignment already happens on master though (see my screenshots).

Reply:
> already happens on master

Ah right, haven't checked against master. I'll let @KKoukiou to decide if she wants to fix it as part of this PR or if we are going to do it as followup.

Reply:
@marusak @martinpitt I will post follow up (if I can figure it out) or open an issue (if not).

Reply:
https://github.com/cockpit-project/cockpit/issues/15511